bridge/bridgeapp/forms.py

```diff
@@ -10,24 +10,27 @@ class ResponseForm(forms.Form):
         required=True,
         label='',
         widget=forms.Textarea(attrs={
-            'rows': '2',
+            'rows': '3',
             'cols': '50',
-            'placeholder': 'Write a response here.'
-            }))
+            'autofocus': True,
+            'placeholder': 'Write a response here...'
+        }))
 
 
 class ThreadForm(forms.Form):
     body = forms.CharField(
         required=True,
         label='',
         widget=forms.Textarea(attrs={
-            'rows': '2',
+            'rows': '3',
             'cols': '50',
-            'placeholder': 'Post a question here.'
-            }))
+            'autofocus': True,
```

Comment:
Can you give example of a site that auto-focus input fields other than on the login/sign up page?